decidim-core/lib/decidim/core/api/author_interface.rb

```diff
@@ -8,5 +8,10 @@ module Decidim
 
     field :name, !types.String, "The author's name"
     field :avatarUrl, !types.String, "The author's avatar url"
+    field :isVerified, !types.Boolean, "Whether the author is verified or not" do
```

Comment:
Instead of the block, you can use `property: :verified?`. This will automatically call `verified?` on the given object, so code is cleaner :)